src/test/resources/com/puppycrawl/tools/checkstyle/checks/regexp/regexp/InputRegexpTrailingComment11.java

```diff
@@ -1,18 +1,14 @@
 /*
 Regexp
 format = don't use trailing comments
-message = (default)null
-illegalPattern = (default)false
-duplicateLimit = (default)0
-errorLimit = (default)100
```

Comment:
Keep defaults

Reply:
I removed them to make the violation comment closer to the line of violation. In the new changes I tried to put the violation comment in the first line of config comment, please let me know if that works.

Reply:
Such lines has to stay.
If no other way to bring it closer, I will recall my request

Reply:
Sure, let's see if CI is green with the new changes. If not, we can try the older way to bring the comment closer.

Reply:
It looks like violation triggers on config.
Can you try to make:
`format = don't\ use\ trailing\ comments`

It is absolutely same for regexp, but it will not trigger, violation on config

Reply:
Done